CMakeLists.txt

```diff
@@ -338,3 +384,23 @@ if(NOT DEFINED SNMALLOC_ONLY_HEADER_LIBRARY)
 
   clangformat_targets()
 endif()
+
+install(TARGETS snmalloc EXPORT snmallocConfig)
```

Comment:
Should we be installing the `.so`s?

Reply:
We are, on line 290 - they're installed as a side effect of installing the library target.  We only need to install the headers separately because snmalloc has an unusual directory structure that isn't supported by cmake.